specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/workspace.json

```diff
@@ -935,6 +935,55 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Synapse/workspaces/{workspaceName}/scopePools": {
+      "get": {
+        "tags": [
+          "ScopePools"
+        ],
+        "operationId": "ScopePoolsListByWorkspace_Get",
```

Comment:
What's the difference between these two PRs?
https://github.com/Azure/azure-rest-api-specs/pull/20319